doc/source/security/using-kmip.rst

```diff
@@ -0,0 +1,78 @@
+.. _using-kmip:
+
+==========================================================
+Using the Key Management Interoperability Protocol (KMIP)
+==========================================================
+
+This feature is **technical preview** quality.
+
+**Percona Server for MySQL** 8.0.27-18 adds support for Key Management Interoperability Protocol (KMIP).
```

Comment:
The full name is "OASIS Key Management Interoperability Protocol" - how about adding a link to the spec? http://www.oasis-open.org/committees/kmip/